test/e2e-v2/cases/win/e2e.yaml

```diff
@@ -25,8 +25,8 @@ setup:
       command: export PATH=/tmp/skywalking-infra-e2e/bin:$PATH
     - name: install yq
       command: bash test/e2e-v2/script/prepare/setup-e2e-shell/install.sh yq
-    - name: install swctl
-      command: bash test/e2e-v2/script/prepare/setup-e2e-shell/install.sh swctl
+#    - name: install swctl
+#      command: bash test/e2e-v2/script/prepare/setup-e2e-shell/install.sh swctl
 
```

Comment:
I think this changes for your local test, should rollback

Reply:
revert.